org.eclipse.scanning.test/src/org/eclipse/scanning/test/event/queues/spooler/SpoolerServiceTest.java

```diff
@@ -0,0 +1,7 @@
+package org.eclipse.scanning.test.event.queues.spooler;
+
+public class SpoolerServiceTest {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty class, write its code or make it an "interface". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2094)